### PR TITLE
Add styling to Bookstore

### PR DESCRIPTION
**Pull request by fernando-silvabr66**
Update project according to Zeplin platform (project requirement).

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -2,16 +2,27 @@ import React from 'react';
...
-          <Link to="/Bookstore/books">Books</Link>
-        </li>
-        <li>
-          <Link to="/Bookstore/categories">Categories</Link>
-        </li>
-      </ul>
+  <header className="container-fluid">
+    <nav className="navbar navbar-expand-lg navbar-light bg-white p-5">
+      <div className="container">
+        <h1 className="display-4">BookStore CMS</h1>
```
Comment (SajjadAhmad14) [inline src/components/Header.jsx:9]: Please add more margin between these three headers of the navigation as can be seen in the design.
margin

File: src/components/Header.jsx [hunk 2]
```diff
@@ -2,16 +2,27 @@ import React from 'react';
...
 
 const Header = () => (
-  <header>
-    <nav>
-      <ul>
-        <li>
-          <Link to="/Bookstore/books">Books</Link>
-        </li>
-        <li>
-          <Link to="/Bookstore/categories">Categories</Link>
```
Comment (SajjadAhmad14) [inline src/components/Header.jsx:12]: Please use font color for `Books`, and `Categories` which is `#121212`, and also use the same font size as can be seen in the design.
head

File: src/components/Header.jsx [hunk 3]
```diff
@@ -2,16 +2,27 @@ import React from 'react';
...
+              </Link>
+            </li>
+            <li className="nav-item">
+              <Link to="/Bookstore/categories" className="nav-link">Categories</Link>
+            </li>
+          </ul>
+        </div>
+        <div>
+          <i className="fa fa-user"> </i>
+        </div>
```
Comment (SajjadAhmad14) [inline src/components/Header.jsx:22]: Please use the same color for the user icon which is `#0290ff`. Kindly follow the design for the icon and its color.
icon

File: src/components/BookItem.jsx [hunk 4]
```diff
@@ -11,20 +14,55 @@ const BookItem = (props) => {
...
   const dispatch = useDispatch();
 
   const removeHandler = () => {
     dispatch(deleteBook(id));
   };
 
   return (
-    <li>
-      <span>{category}</span>
-      <h3>{title}</h3>
```
Comment (SajjadAhmad14) [inline src/components/BookItem.jsx:23]: Please use `#121212` color for the book title and the same font size as defined in the design for this section.
title

File: src/components/AddBook.jsx [hunk 5]
```diff
@@ -23,34 +24,39 @@ const AddBook = () => {
...
+            (e) => setCategory(e.target.value)
+          }
+        >
+          <option value="action">Action</option>
+          <option value="science fiction">Science Fiction</option>
+          <option value="economy">Economy</option>
+          <option value="technology">Technology</option>
+          <option value="games">Games</option>
+        </select>
+        <button className="add-btn" type="submit">Add Book</button>
```
Comment (SajjadAhmad14) [inline src/components/AddBook.jsx:57]: Please position this button at the end of the form horizontally. Kindly follow the design for this section.
btn

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Fernando, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/AddBook.jsx (hunk 5 above)
Comment (fernando-silvabr66) [reply src/components/AddBook.jsx:57]: Done!

File: src/components/Header.jsx (hunk 3 above)
Comment (fernando-silvabr66) [reply src/components/Header.jsx:22]: Color: Done!

File: src/components/BookItem.jsx (hunk 4 above)
Comment (fernando-silvabr66) [reply src/components/BookItem.jsx:23]: Done!

File: src/components/Header.jsx (hunk 2 above)
Comment (fernando-silvabr66) [reply src/components/Header.jsx:12]: Done!

**Review by nicupop729 — APPROVED** [review]
<h1>Status: Approved 🥇 ✔</h1>
Hi @fernando-silvabr66 ,

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**